README.rst

```diff
@@ -35,21 +35,83 @@ which can then be used like any Django plugin.
 Intructions on how to add the OMERO.iviewer app to your installed OMERO.web apps
 can be found in the `OMERO.iviewer README <plugin/omero_iviewer/README.rst>`_.
 
-**Note**: 
+**Note**:
 
 Should you like or need to rebuild OMERO.iviewer's internal ol3 viewer,
 please read the section *ol3-viewer* below!
 
 Development
 ===========
 
-Run:
+Building the uncompressed version of iviewer as described above the application
+can be deployed locally or to a test server, to then be debugged.
+It is also possible to run/debug it with webpack dev-server:
 
 ::
- 
+
     $ npm run dev
 
-To be able to run the ``webpack-dev`` server locally at: ``localhost:3000``
+will build the bundle and start the webpack dev-server (localhost:8080)
+
+Either development setup expects the following:
+- an OMERO server installation
```

Comment:
the bullet list is not correctly rendered on gh.
I think you need to add a line before the first item